docker/Dockerfile

```diff
@@ -5,12 +5,14 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     && rm -rf /var/lib/apt/lists/*
 
 RUN python3 -m pip install torch==1.13.1+cu117 torchvision==0.14.1+cu117 --extra-index-url https://download.pytorch.org/whl/cu117
-RUN python3 -m pip install sentencepiece cmake transformers<4.30.0 protobuf==3.20.3 pybind11 mmengine
+RUN python3 -m pip install sentencepiece cmake "transformers<4.30.0" protobuf==3.20.3 pybind11 mmengine
```

Comment:
transformers<4.30.0 is no longer required. 
all the packages are listed in `requirements.txt` and can be installed through in the following `pip install .`

Reply:
no protobuf and cmake in https://github.com/InternLM/lmdeploy/blob/main/requirements.txt

Reply:
`protobuf` can be installed through `tritonclient[all]`